bigtable/google/cloud/bigtable/table.py

```diff
@@ -618,6 +658,29 @@ def _do_mutate_retryable_rows(self):
         return self.responses_statuses
 
 
+class _ClusterStatePB(object):
+    """Map cluster state int value to string value.
+            :rtype: _ClusterStatePB object
```

Comment:
Please add description for `:type replication_state: int` `:param replication_state:`
Please move [`possible replication state values`](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/2e418e7a786c71bc980729335f8ee1c39f44fbae/bigtable/google/cloud/bigtable/table.py#L266-L285) here.
Please show `int` values to the corresponding `ReplicationState` `string` values. Something like
`int 0 is STATE_NOT_KNOWN (str) ...`